contracts/src/PBHVerifier.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {WorldIDProxy} from "@world-id-contracts/abstract/WorldIDProxy.sol";
+
+/// @title PBH Verifier
+/// @author Worldcoin
+/// @notice An implementation of an on chain PBH verifier.
+contract PBHVerifier is WorldIDProxy {
```

Comment:
Small nit, but maybe we just inherit `ERC1967Proxy` here instead of `WorldIDProxy`.  Should be the same thing, just less gas